### PR TITLE
Adds a Familiar Hat for dressup

### PR DESCRIPTION
**Pull request by Mithrandalf**
**What does this PR do:**
Adds a new hat to the autodrobe, one which may be familiar to any fans of large posing anime men.

**Images of sprite/map changes (IF APPLICABLE):**
boom
image
image
image

**Changelog:**
:cl: name here
add: new hat in the autodrobe
/:cl:

it didnt work last time so I did it again

### CODE REVIEW

**Comment by Mithrandalf** [comment]
I do not understand this one at all
image
image

File: paradise.dme [hunk 1]
```diff
@@ -611,7 +611,6 @@
 #include "code\game\machinery\shieldgen.dm"
 #include "code\game\machinery\Sleeper.dm"
 #include "code\game\machinery\slotmachine.dm"
-#include "code\game\machinery\snow_machine.dm"
```
Comment (Ty-Omaha) [inline paradise.dme]: you need to make sure your thing is up to date and make sure these files have a checkmark when you compile

File: paradise.dme [hunk 2]
```diff
@@ -747,7 +746,6 @@
 #include "code\game\objects\effects\misc.dm"
 #include "code\game\objects\effects\overlays.dm"
 #include "code\game\objects\effects\portals.dm"
-#include "code\game\objects\effects\snowcloud.dm"
```
Comment (Ty-Omaha) [inline paradise.dme]: same

File: paradise.dme (hunk 1 above)
Comment (Ty-Omaha) [reply paradise.dme]: image

example

**Comment by Mithrandalf** [comment]
ah alright good shit I'll see if I managed to click those fucking boxes by mistake tommorow

**Comment by Birdtalon** [comment]
Is it just me but the hat looks different in pic number 2?

**Comment by Mithrandalf** [comment]
well yeah pic 2 is the inhand sprite and its larger

**Comment by Mithrandalf** [comment]
That's fair. I did the worn sprite well before the inhand.

I'll change the texture, one sec.

**Comment by taukausanake** [comment]
Seems like something more akin to a fluff item. Just a retextured top hat or something

**Comment by Ty-Omaha** [comment]
>Seems like something more akin to a fluff item. Just a retextured top hat or something

I guess we only need 1 jumpsuit color then

**Comment by KasparoVy** [comment]
Yeah, grey

**Comment by Mithrandalf** [comment]
This is entirely a fluff item, purely something to throw into the autodrobe so 1/100 clowns can decide to wear it if they want.

Mostly though, this is a way for me to re-learn how to add stuff in github and the hat itself is just an excuse for me to do it.

**Comment by Mithrandalf** [comment]
holy shit lads I think it worked

**Comment by Mithrandalf** [comment]
nope hold on

**Comment by Mithrandalf** [comment]
Updated the pics w/ new design. Everything is working and it only took me a little over 2 days.

**Comment by variableundefined** [comment]
It conflicts

**Comment by Mithrandalf** [comment]
Did that fix it?

**Pull request merged**